### PR TITLE
[OTE v0.5.0][rc1] Update changelog for v0.5.0-rc1 release

### PR DESCRIPTION
**Pull request by chuneuny-emily**
Update changelog for v0.5.0-rc1 release

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,27 @@
...
 All notable changes to this project will be documented in this file.
 
+## \[v0.5.0\]
+
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
+- Integrate ov-telemetr (<https://github.com/openvinotoolkit/training_extensions/pull/1568>)
```
Comment (JihwanEom) [inline CHANGELOG.md]: A simple typo? `ov-telemetr` -> `ov-telemetry`

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,27 @@
...
+
+### Changed
+
+- Update OpenVINO to 2022.3 release (<https://github.com/openvinotoolkit/training_extensions/pull/1393>)
+
+### Fixed
+
+- h-label head output fix (<https://github.com/openvinotoolkit/training_extensions/pull/1458>)
```
Comment (goodsong81) [inline CHANGELOG.md]:
```suggestion
- Fix h-label head output bug in OV inference (<https://github.com/openvinotoolkit/training_extensions/pull/1458>)
```

File: CHANGELOG.md (hunk 1 above)
Comment (chuneuny-emily) [reply CHANGELOG.md]: > A simple typo? `ov-telemetr` -> `ov-telemetry`
Sorry. I fixed it. Thanks

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,27 @@
 
 All notable changes to this project will be documented in this file.
 
+## \[v0.5.0\]
+
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
```
Comment (goodsong81) [inline CHANGELOG.md]: Let's remove MPA. Maybe we can just move-left 2 "Add" items?

File: CHANGELOG.md [hunk 4]
```diff
@@ -2,6 +2,27 @@
...
+
+- Update OpenVINO to 2022.3 release (<https://github.com/openvinotoolkit/training_extensions/pull/1393>)
+
+### Fixed
+
+- h-label head output fix (<https://github.com/openvinotoolkit/training_extensions/pull/1458>)
+- Fixing deprecated np.bool issue from numpy==1.24.0 (<https://github.com/openvinotoolkit/training_extensions/pull/1455>)
+- Fixing tiling OpenVINO backward compatibility (<https://github.com/openvinotoolkit/training_extensions/pull/1516>)
+- Fixing indexing in hierarchical classification inference (<https://github.com/openvinotoolkit/training_extensions/pull/1551>)
+- Some minor issues
```
Comment (goodsong81) [inline CHANGELOG.md:24]: Please add
- Copy feature vector to resolve duplication issue (#1511)

File: CHANGELOG.md [hunk 5]
```diff
@@ -2,6 +2,27 @@
...
+### Added
+
+- Model Preparation Algorithm (MPA)
+  - Add tiling in rotated detection (<https://github.com/openvinotoolkit/training_extensions/pull/1420>)
+  - Add Cythonize AugMixAugment (<https://github.com/openvinotoolkit/training_extensions/pull/1478/>)
+- Integrate ov-telemetry (<https://github.com/openvinotoolkit/training_extensions/pull/1568>)
+
+### Changed
+
+- Update OpenVINO to 2022.3 release (<https://github.com/openvinotoolkit/training_extensions/pull/1393>)
```
Comment (sovrasov) [inline CHANGELOG.md]: In this PR NNCF has been updated as well to the pre-2.4 version

**Review by goodsong81 — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 5 above)
Comment (chuneuny-emily) [reply CHANGELOG.md]: > In this PR NNCF has been updated as well to the pre-2.4 version

Updated.

File: CHANGELOG.md (hunk 4 above)
Comment (chuneuny-emily) [reply CHANGELOG.md:24]: > Please add
> 
> * Copy feature vector to resolve duplication issue ([[OTE][dev] Copy Feature Vector #1511](https://github.com/openvinotoolkit/training_extensions/pull/1511))

Added

File: CHANGELOG.md (hunk 3 above)
Comment (chuneuny-emily) [reply CHANGELOG.md]: Removed MPA category

File: CHANGELOG.md (hunk 2 above)
Comment (chuneuny-emily) [reply CHANGELOG.md]: > 

Updated

**Review by goodsong81 — APPROVED** [review]
LGTM! Thank you for the detailed update.

**Review by sovrasov — APPROVED** [review]

**Pull request merged**